bin/compile

```diff
@@ -147,7 +147,7 @@ export YARN2_PATH
 if [[ "$YARN2" == "true" ]]; then
   if [[ "$NODE_MODULES_CACHE" == "true" ]]; then
     warn "
-      WARNING- You are using Yarn 2. The buildpack won't cache because Heroku is expecting dependencies from the .yarn directory for each build. To unset this and other similar environment variables, read here: https://devcenter.heroku.com/articles/migrating-to-yarn-2#change-environment-variables
+      WARNING- Yarn 2 does not support the --production flag, so yarn install will be run without the flag. More info about yarn install flags here: https://yarnpkg.com/cli/install
```

Comment:
I don't necessarily know how to implement, but Yarn 2 does support a --production flag when working with monorepos with https://yarnpkg.com/cli/workspaces/focus

I think maybe there is a way we can do something like add monorepo support to this buildpack and if a user passes in a `APP_WORKSPACE` or something we can run that focus --production.

Reply:
Yeah - this message will need to be reworded because the `--production` flag isn't what it used to be for an entire project while simultaneously explaining that it's available for a workspace setup. 👍